src/app/api/v1/scan/route.ts

```diff
@@ -11,6 +11,12 @@ import { getL10n } from "../../../functions/server/l10n";
 import { getBreachLogo } from "../../../../utils/breachLogo";
 import { Breach } from "../../../(nextjs_migration)/(authenticated)/user/breaches/breaches";
 
+/**
+ * @typedef {{ success: false }} RequestBreachScanErrorResponse
```

Comment:
Since this is a .ts file I think you can specify interfaces / types vs. doing it in JSDoc

Reply:
ah yes good call, i just copy/pasted